libbpf-tools/tcpretrans.bpf.c

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2020 Anton Protopopov
+// Copyright (c) 2021 Red Hat
+//
+// Based on tcpconnect.c by Anton Protopopov and
+// tcpretrans(8) from BCC by Brendan Gregg
+#include <vmlinux.h>
+
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+
+#include "tcpretrans.h"
+
+/* Define here, because there are conflicts with include files */
+#define AF_INET		2
+#define AF_INET6	10
```

Comment:
weird indention.

Reply:
I think this is a rendering problem with this diff format.  It looks good here and locally: https://github.com/iovisor/bcc/blob/b709dc49c65d2d651a93e66a6451e520955ba09b/libbpf-tools/tcpretrans.bpf.c

Reply:
You are right.